genome_grist/copy_local_genomes.py

```diff
@@ -50,15 +57,28 @@ def main():
 
         print(f"read identifer '{ident}' and name '{remainder}'")
 
-        destfile = os.path.join(args.output_directory, f"{ident}_genomic.fna.gz")
-
         is_gzipped = False
         with contextlib.suppress(OSError):
             with gzip.open(filename) as fp:
                 fp.read(1)
                 is_gzipped = True
 
-        if is_gzipped:
+        destfile = os.path.join(args.output_directory, f"{ident}_genomic.fna.gz")
+        
+        
+        if args.sym and not is_gzipped:
+            print(f"--sym option requires the Fasta files to be gzipped first.", file= sys.stderr)
+            sys.exit(1)
```

Comment:
```suggestion
            sys.exit(-1)
```

I usually use -1 for some reason ;)